src/utils/mesh_client.ts

```diff
@@ -1,45 +1,74 @@
 import { SignedOrder, ValidationResults, WSClient, WSOpts } from '@0x/mesh-rpc-client';
 import Axios from 'axios';
 import * as _ from 'lodash';
+import * as rax from 'retry-axios';
 
 import { MESH_ORDERS_BATCH_SIZE } from '../constants';
+import { logger } from '../logger';
 
 import { utils } from './utils';
 
+// Attach retry-axios to the global instance
+rax.attach();
```

Comment:
Can we make a `utils/axios` file and import axios from there and include this statement in there as well?

Reply:
Like we do for logger.